module/tools/SystemConfiguration/data/system/default/etc/pacman.conf

```diff
@@ -0,0 +1,19 @@
+[options]
```

Comment:
Wait, this file shouldn't be here. In `module/tools/SystemConfiguration/CMakeLists.txt` you should have 
```cmake
nuclear_module(DATA_FILES "/etc/pacman.d/mirrorlist:system/default/etc/pacman.d" "/etc/pacman.conf:system/default/etc")
```

This will copy the `pacman.conf` from inside the docker image into teh data files for `SystemConfiguration`. This will then install result in the docker image's `pacman.conf` being installed on to the device

Reply:
Alright, thanks. That makes sense :+1: